api/etcdserverpb/rpc.proto

```diff
@@ -834,6 +834,9 @@ message WatchResponse {
   // framgment is true if large watch response was split over multiple responses.
   bool fragment = 7 [(versionpb.etcd_version_field)="3.4"];
 
+  // retry is true if watcher should be retried on cancel.
+  bool retry = 8 [(versionpb.etcd_version_field)="3.5"];
```

Comment:
I'll backport to 3.5, so setting version 3.5 here

Reply:
No backports of fields!